### ISSUE TITLE
Ansible ServiceInstance CRD colliding with Service Catalog

### ISSUE THREAD

**Issue by jboyd01** [openshift/ansible-service-broker#914]
## Bug:
When I attempt to work with Catalog service instances from the openshift OC command line, things appear to get confused with ASB's ServiceInstance:
```console
$ oc get serviceinstances -o yaml
apiVersion: v1
items:
- apiVersion: automationbroker.io/v1
  kind: ServiceInstance
  metadata:
    clusterName: ""
    creationTimestamp: 2018-04-30T14:29:46Z
    generation: 1
    name: ee039f97-4c82-11e8-ba53-0242ac110009
    namespace: ansible-service-broker
    resourceVersion: "4304"
    selfLink: /apis/automationbroker.io/v1/namespaces/ansible-service-broker/serviceinstances/ee039f97-4c82-11e8-ba53-0242ac110009
    uid: ee5709ff-4c82-11e8-8c3a-c85b76aab8ec
  spec:
    bindingIDs: []
    bundleID: ddd528762894b277001df310a126d5ad
    context:
      namespace: jay
      plateform: kubernetes
    parameters: '{"_apb_last_requesting_user":"dev","_apb_plan_id":"dev","_apb_service_class_id":"ddd528762894b277001df310a126d5ad","_apb_service_instance_id":"ee039f97-4c82-11e8-ba53-0242ac110009","mysql_database":"devel","mysql_password":"xx","mysql_user":"devel","mysql_version":"5.7","service_name":"mysql"}'
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
```
**What happened**:
`oc get/edit` gets/edits ASB's service instances, not catalog's.

**What you expected to happen**:
I expect to `$ oc get serviceinstances` to reference Service Catalog instances, not Ansibile's.

**How to reproduce it**:
oc cluster up  followed by run_latest_build.sh to install ASB.

**Comment by djzager** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1566924

And fixed in https://github.com/openshift/ansible-service-broker/pull/907

**Comment by djzager** [comment]
This is also released in `ansible-service-broker-1.2.8` which can be found in the latest ansible-service-broker image released in the past hour. Verify with the latest broker container image and report back, if you could, please.
